app/models/order.rb

```diff
@@ -80,10 +80,10 @@ def business_days_after num
 
     def delivery_day_limit
       errors.add :delivery_day, :blank and return if delivery_day.blank?
-      if delivery_day < min_day or delivery_day > max_day or [0, 6].include?(delivery_day.wday) 
-        errors.add(:delivery_day, 
-          I18n.t('activerecord.errors.messages.invalid_business_day', 
-            from: BUSINESS_DAY_FROM, to: BUSINESS_DAY_TO)) 
+      if delivery_day < min_day or delivery_day > max_day or [0, 6].include?(delivery_day.wday)
+        errors.add(:delivery_day,
+          I18n.t('activerecord.errors.messages.invalid_business_day',
```

Comment:
[rubocop] Align the parameters of a method call if they span more than one line.